### ISSUE TITLE
Can we add PlanWorkspacePath() method?

### ISSUE THREAD

**Issue by DavidB-CMU** [personalrobotics/or_trajopt#19]
Can we add a PlanWorkspacePath() method, or would that be slow?

**Comment by siddhss5** [comment]
Someone has to write it ... or are you referring to Rachel's code?

**Comment by DavidB-CMU** [comment]
Yes Rachel and I were wondering if Trajopt was capable of taking an input of a desired workspace path, and if it would be good at creating such a path?  
Currently we are only using Trajopt to optimize a configuration space path.
